### ISSUE TITLE
[UI] [Accessibility] Employer user is unable to access update and delete buttons using only keyboard

### ISSUE THREAD

**Issue by miguelbravo09** [miguelbravo09/benefits-dashboard-app#4]
## **Description**
Employer user is unable to access update and delete buttons using only the keyboard. 

## **Steps to Reproduce**
1. Log in as employer
2. Using only the keyboard try to navigate to update button

## **Expected Behavior**
User is able to focus on update button or delete button

## **Actual Behavior**
User is unable to focus on update button or delete button.

## **Environment**
- **Browser:** Edge 129.0.2792.79
- **OS:** Windows 11
- **Screen Resolution:** 1920x1080
- **Device:** Laptop
- **Version:** 1.0

## **Screenshots**
https://github.com/user-attachments/assets/de8e8891-35f9-47bd-beca-82487e033d17
